### PR TITLE
Vv 67 trust manager flow trust managers

### PR DESCRIPTION
**Pull request by nax2uk**
# What
Added the functionality to view,edit and delete organisation managers.

# Why
New Feature Request
# Screenshots
image

# Notes
Will create gateway and relevant contract and unit tests for managers in another ticket.

### CODE REVIEW

**Review by miranda-hawkes — CHANGES_REQUESTED** [review]
Rest of review pending

File: pages/api/archive-manager.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import withContainer from "../../src/middleware/withContainer";
+
+export default withContainer(
+  async ({ headers, body, method }, res, { container }) => {
+    if (method !== "DELETE") {
+      res.status(405);
+      res.end();
+      return;
+    }
```
[the remark this answers is not in the document]
Comment (nax2uk) [reply pages/api/archive-manager.js]: Good stuff! I made the changes and pushed it up.

File: pages/api/archive-manager.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    const trustAdminIsAuthenticated = container.getTrustAdminIsAuthenticated();
+
+    const trustAdminAuthenticatedToken = trustAdminIsAuthenticated(
+      headers.cookie
+    );
+    if (!trustAdminAuthenticatedToken) {
+      res.status(401);
+      res.end();
+      return;
+    }
```
[the remark this answers is not in the document]
Comment (nax2uk) [reply pages/api/archive-manager.js]: Good stuff. I have updated and pushed it up. 😀

**Review by miranda-hawkes — APPROVED** [review]
Nice one, looks really great

File: src/components/ManagerForm/index.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  const action = manager.uuid ? "Edit" : "Add";
+
+  const hasError = (field) =>
+    errors.find((error) => error.id === `${field}-error`);
+
+  const errorMessage = (field) => {
+    const error = errors.filter((err) => err.id === `${field}-error`);
+    return error.length === 1 ? error[0].message : "";
+  };
```
Comment (miranda-hawkes) [inline src/components/ManagerForm/index.js:20]: These methods have also been extracted into common helpers in `src/helpers/pageErrorHandler`, could you use them instead please?

**Review by miranda-hawkes — CHANGES_REQUESTED** [review]

File: src/components/ManagerForm/index.js (hunk 3 above)
Comment (nax2uk) [reply src/components/ManagerForm/index.js]: I just removed validation error checking in current ManagerForm as it only consists of a select component which has fixed dropdown values that does not need validating. Will not be using the error helper functions for this particular component.

**Review by miranda-hawkes — APPROVED** [review]

**Pull request merged**